.github/workflows/compat-check-latest-latest.yml

```diff
@@ -1,19 +1,32 @@
 name: Compat Check w/PL - latest/latest
 
 on:
+  pull_request:
   schedule:
-    - cron: "0 3 * * 1-5"  # Run daily at 3am Mon-Fri
+    - cron: "0 0 * * 1-5"  # Run daily at 0am Mon-Fri
```

Comment:
Are we updating that too?

Reply:
Since this is getting heavy, I would like to start each compat workflow 1 hour apart from 0am to 3am.